web/src/echo.js

```diff
@@ -270,30 +269,6 @@ export function edit_locally(message, request) {
     const raw_content = request.raw_content;
     const message_content_edited = raw_content !== undefined && message.raw_content !== raw_content;
 
-    if (request.new_topic !== undefined || request.new_stream_id !== undefined) {
-        const new_stream_id = request.new_stream_id;
-        const new_topic = request.new_topic;
-        stream_topic_history.remove_messages({
-            stream_id: message.stream_id,
-            topic_name: message.topic,
-            num_messages: 1,
-            max_removed_msg_id: message.id,
-        });
-
-        if (new_stream_id !== undefined) {
-            message.stream_id = new_stream_id;
-        }
-        if (new_topic !== undefined) {
-            message.topic = new_topic;
-        }
-
-        stream_topic_history.add_message({
-            stream_id: message.stream_id,
-            topic_name: message.topic,
-            message_id: message.id,
-        });
-    }
```

Comment:
Can you extend the commit message for why you're removing the local echo for this? Is it buggy, not important from a latency standpoint, or something else?

Reply:
Added some explaination in commit message. Basically we try to locally echo the move for locally echoed messages but we did not allow the user to move the locally echoed messages.